mne/channels/montage.py

```diff
@@ -699,6 +729,7 @@ def _set_montage(info, montage, match_case=True, match_alias=False,
 
     if isinstance(montage, DigMontage):
         mnt_head = _get_montage_in_head(montage)
+        del montage
```

Comment:
Is that what you meant @larsoner?

Reply:
yes, sorry for the (very) slow response!